src/plugins/ssh/index.ts

```diff
@@ -63,6 +63,7 @@ const DEFAULT_VALIDATION_WINDOW_MS = 5e3;
  * Note that each attempt consumes ~ 1 s.
  */
 const MAX_SSH_RETRIES = 30;
```

Comment:
If we differ between AWS and GCP we should indicate that this constant is for AWS.

Reply:
This is a default. I will rename it to `DEFAULT_MAX_SSH_RETRIES`